source/glossary.rst

```diff
@@ -144,6 +144,27 @@ and another geared toward developers. You can find the
    qzv
      See :term:`visualization`.
 
+   rarefy
+     A technique to minimize uneven sampling depth by subsampling without 
+     replacement so that all :term: `samples <sample>` have the same sampling
```

Comment:
```suggestion
     A technique to remove uneven sampling depth by subsampling without 
     replacement so that all :term: `samples <sample>` have the same sampling
```